src/toil/resource.py

```diff
@@ -557,35 +578,35 @@ def _resourcePath(self):
             return self.dirPath
 
     @classmethod
-    def _initModuleName(cls, dirPath):
+    def _initModuleName(cls, dirPath: str) -> Optional[str]:
         for name in ('__init__.py', '__init__.pyc', '__init__.pyo'):
             if os.path.exists(os.path.join(dirPath, name)):
                 return name
         return None
 
-    def _rootPackage(self):
+    def _rootPackage(self) -> Any:
```

Comment:
Doesn't this always return `str`? Since `self.name` would be a `str` if it is being split?